build/run-tests.sh

```diff
@@ -5,4 +5,4 @@ KUBECONFIG=${1-openshift.local.clusterup/openshift-apiserver/admin.kubeconfig}
 echo "Using KUBECONFIG '$KUBECONFIG'"
 
 go clean -testcache ./test/e2e
-go test -v ./test/e2e -timeout 45m
+go test -v ./test/e2e -run TestExternalServiceWithAuth -timeout 45m
```

Comment:
This should be omitted from the PR so the CI doesn't run only one test.

Reply:
Yes, sure. But we need to decide which set of the e2e test will keep for CI. What do you think about these?

Reply:
With unit tests in place we can get rid off some e2e execution heavy duplicates and keep executing all in e2e. See my suggestion:

**Remove:** TestSimple, TestNodeStartup, TestClusterFormationWithTls, TestContainerCPUUpdate, TestContainerMemoryUpdate, TestContainerJavaOptsUpdate

**Keep:** TestClusterFormation, TestCacheService, TestPernamentCache, TestCheckDataSurviveToShutdown, TestExternalService, TestExternalServiceWithAuth

Basically we can remove all the `*Update` tests. As far as we are concerned they have equivalent unit test at this moment. `TestSimple` isn't a test. `TestNodeStartup` is subtest of `TestClusterFormation` and `TestClusterFormationWithTLS` doesn't provide any more assurance as clustering uses different protocol meaning that TLS enabled doesn't affect it.

@rigazilla What do you think about this?

Reply:
Thanks for response, @Crumby 
In case of the issue #311 we should need to keep one of the *Update tests with replicas=2 or create a new one. In other cases, the list looks very good.